be/src/runtime/load_path_mgr.cpp

```diff
@@ -91,8 +88,8 @@ Status LoadPathMgr::init() {
 
 Status LoadPathMgr::allocate_dir(const std::string& db, const std::string& label,
                                  std::string* prefix) {
-    if (_path_vec.empty()) {
-        return Status::InternalError("No load path configured.");
+    for (auto& store_path : _exec_env->store_paths()) {
+        _path_vec.push_back(store_path.path + "/" + MINI_PREFIX);
     }
```

Comment:
The size of _path_vec increases multi times.

Reply:
done